src/app/datalogger/flash_dumper.c

```diff
@@ -28,8 +28,52 @@
 
 static void flash_dumper_periodic_flush(void *data);
 
+#define MAX_FNAME_L (128)
+
+static const char *makeFileName() {
+    const char *counter_fname = "filenumber.dat";
```

Comment:
i think you want to make this filenum.dat so it fits in 8+3

Reply:
yup